2007-01-08  Daniel Jacobowitz  <dan@codesourcery.com>

	* sysdeps/mips/ldsodefs.h: Correct multiple inclusion guard.

--- ports/sysdeps/mips/ldsodefs.h.orig	2007-04-20 01:07:43.000000000 +0200
+++ ports/sysdeps/mips/ldsodefs.h	2007-04-20 01:07:55.000000000 +0200
@@ -17,7 +17,7 @@
    Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
    02111-1307 USA.  */
 
-#ifndef __LDSODEFS_H \
+#ifndef __LDSODEFS_H
 #define _MIPS_LDSODEFS_H 1
 
 #include <elf.h>
